### PR TITLE
[Obmin-265] Replaced Button component with component from Storybook

### PR DESCRIPTION
**Pull request by Mish-hash**

### CODE REVIEW

File: src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import { useCallback } from 'react';
-
 import { CheckSvg } from 'assets/icons';
 
 import { Div, LabelSquare, Label, Input } from './styles';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx:4]: import * as Styles from './styles'

File: src/main/resources/react/src/pages/FormRegister/sign-in/index.jsx [hunk 2]
```diff
@@ -3,13 +3,14 @@ import * as yup from 'yup';
...
 import { useNavigate, useLocation } from 'react-router-dom';
+import { Button } from '@wolshebnik/obminyashka-components';
 
 import { NO_SPACE } from 'config';
 import { GoogleSvg } from 'assets/icons';
 import { route } from 'routes/routeConstants';
 import { putUserThunk } from 'store/auth/thunk';
 import { getTranslatedText } from 'components/local/localization';
-import { CheckBox, Button, InputForAuth } from 'components/common';
+import { CheckBox, InputForAuth } from 'components/common';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/FormRegister/sign-in/index.jsx:13]: sort

File: src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx [hunk 3]
```diff
@@ -1,13 +1,14 @@
 import { useState } from 'react';
 import { Formik } from 'formik';
 import { useNavigate } from 'react-router-dom';
+import { Button } from '@wolshebnik/obminyashka-components';
 
 import api from 'REST/Resources';
 import { GoogleSvg } from 'assets/icons';
 import { route } from 'routes/routeConstants';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx:9]: remove space

File: src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
 
 import { getTranslatedText } from 'components/local/localization';
-import { CheckBox, Button, InputForAuth } from 'components/common';
+import { CheckBox, InputForAuth } from 'components/common';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx:11]: sort

File: src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx [hunk 5]
```diff
@@ -1,5 +1,4 @@
 import { useCallback } from 'react';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx:1]: add space

File: src/main/resources/react/src/pages/FormRegister/sign-in/index.jsx (hunk 2 above)
Comment (Mish-hash) [reply src/main/resources/react/src/pages/FormRegister/sign-in/index.jsx:13]: +

File: src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx (hunk 3 above)
Comment (Mish-hash) [reply src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx:9]: changed

File: src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx (hunk 4 above)
Comment (Mish-hash) [reply src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx:11]: changed

File: src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx (hunk 1 above)
Comment (Mish-hash) [reply src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx:4]: changed

File: src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx (hunk 5 above)
Comment (Mish-hash) [reply src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx:1]: changed

File: src/main/resources/react/src/components/common/index.js [hunk 6]
```diff
@@ -1,6 +1,5 @@
 export { BackButton } from './buttons/back-button';
 export { BtnGoodBusiness } from './buttons/btnGoodBusiness';
-export { Button } from './buttons/button';
 export { ButtonAdv } from './buttons/buttonAdv';
 export { ButtonLinkBlue } from './buttons/button-link-blue';
 export { CheckBox } from './checkbox';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/index.js:5]: sort

File: src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx [hunk 7]
```diff
@@ -37,21 +37,21 @@ const FormikCheckBox = ({
...
+      <Styles.Label fontSize={fontSize}>
+        <Styles.Input
           type={type}
           name={name}
           value={value}
           checked={isSelected(value)}
           onChange={() => handleChange(value)}
         />
-        <LabelSquare type={type}>
+        <Styles.LabelSquare type={type}>
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx:49]: space

File: src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx [hunk 8]
```diff
@@ -37,21 +37,21 @@ const FormikCheckBox = ({
...
-        <Input
+    <Styles.Div margin={margin}>
+      <Styles.Label fontSize={fontSize}>
+        <Styles.Input
           type={type}
           name={name}
           value={value}
           checked={isSelected(value)}
           onChange={() => handleChange(value)}
         />
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx:48]: space

File: src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx [hunk 9]
```diff
@@ -85,27 +85,29 @@ const SignUp = () => {
...
+                height={48}
+                lHeight={24}
                 type="submit"
-                width="222px"
-                height="48px"
-                lHeight="24px"
                 isLoading={loading}
+                style={{ marginBottom: 44 }}
                 text={getTranslatedText('auth.signUp')}
                 disabling={!checkbox || !dirty || !isValid || isSubmitting}
```
Comment (ASGladchenko) [inline src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx:95]: In storybook your property disabled here you write disabling. Maybe need change

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/index.jsx [hunk 10]
```diff
@@ -239,14 +239,13 @@ const MySettings = () => {
 
             <Styles.ButtonContainer>
               <Button
+                width={248}
                 type="submit"
-                width="248px"
-                height="49px"
                 isLoading={isFetchEmail}
-                style={{ margin: '50px 0' }}
                 disabling={!isValid && !dirty}
```
Comment (ASGladchenko) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/index.jsx:245]: The same

**Review by ASGladchenko — APPROVED** [review]

**Pull request merged**